docs_user/modules/proc_preparing-block-storage-service-by-customizing-configuration.adoc

```diff
@@ -13,38 +13,18 @@ The high level explanation of the tailor-made approach is:
-in `customServiceConfigSecrets`) at the `cinder: template:` level.
-. Determine if there's any scheduler specific configuration and add it to the
+. Determine what part of the configuration is generic for all the Block Storage services and remove anything that would change when deployed in {OpenShift} ({OpenShiftShort}), such as the `connection` in the `[database]` section, the `transport_url` and `log_dir` in the `[DEFAULT]` sections, and the whole `[coordination]` and `[barbican]` sections. The remaining generic configuration goes into the `customServiceConfig`, or a `Secret` and is then used in `customServiceConfigSecrets`, at the `cinder: template:` level.
+. Determine if there is any scheduler-specific configuration and add it to the
 `customServiceConfig` section in `cinder: template: cinderScheduler`.
-. Determine if there's any API specific configuration and add it to the
+. Determine if there is any API-specific configuration and add it to the
 `customServiceConfig` section in `cinder: template: cinderAPI`.
-. If you have {block_storage} backup deployed, then you get the {block_storage} backup relevant
-configuration options and add them to `customServiceConfig` (or a `Secret` and
-then used in `customServiceConfigSecrets`) at the `cinder: template:
-cinderBackup:` level. You should remove the `host` configuration in the
-`[DEFAULT]` section to facilitate supporting multiple replicas in the future.
-. Determine the individual volume backend configuration for each of the
-drivers. The configuration will not only be the specific driver section, it
-should also include the `[backend_defaults]` section and FC zoning sections is
-they are being used, because the {block_storage} operator doesn't support a
-`customServiceConfig` section global for all volume services.  Each backend
-would have its own section under `cinder: template: cinderVolumes` and the
-configuration would go in `customServiceConfig` (or a `Secret` and then used in
-`customServiceConfigSecrets`).
-. Check if any of the {block_storage} volume drivers being used requires a custom vendor
-image. If they do, find the location of the image in the vendor's instruction
-available in the https://catalog.redhat.com/software/search?target_platforms=Red%20Hat%20OpenStack%20Platform&p=1&functionalCategories=Data%20storage[{rhos_prev_long} {block_storage} ecosystem
-page]
-//kgilliga: Shouldn't this link be this instead? https://catalog.redhat.com/software/search?target_platforms=Red%20Hat%20OpenStack%20Platform&p=1&functionalCategories=Data%20storage&certified_plugin_types=Block%20Storage%20(Cinder)
-and create or modify an `OpenStackVersion` manifest to specify the custom
-image using the key from the `cinderVolumes` section.
+. If {block_storage} backup is deployed, then add the {block_storage} backup configuration options to `customServiceConfig`, or a `Secret` and then used in `customServiceConfigSecrets`, at the `cinder: template:
```

Comment:
If {block_storage} backup is deployed, then add the {block_storage} backup configuration options to `customServiceConfig`, or to a `Secret` that you can add to `customServiceConfigSecrets`, at the `cinder: template: cinderBackup:` level.